downstream/modules/platform/ref-eda-controller-variables.adoc

```diff
@@ -0,0 +1,44 @@
+
+[id="event-driven-ansible-controller_{context}"]
+= Event-Driven Ansible controller variables
+
+[cols="50%,50%",options="header"]
+|====
+| *Variable* | *Description* 
+| *`automationedacontroller_admin_username`* | Username used by django to identify and create the admin superuser in {EDAcontroller}.
+
+Default = admin
+| *`automationedacontroller_admin_email`* | Email address used by django for the admin user for {EDAcontroller}. 
+
+Default = admin@example.com
+| *`automationedacontroller_disable_https`* | Boolean flag to disable HTTPS {EDAcontroller}. 
+
+Default = false
+| *`automationedacontroller_disable_hsts`* | Boolean flag to disable HSTS {EDAcontroller}. 
+
+Default = false
+| *`automationedacontroller_user_headers`* | List of additional nginx headers to add to {EDAcontroller}'s nginx configuration. 
+
+Default = empty list
+| *`automationedacontroller_nginx_tls_files_remote`* | Boolean flag to specify whether cert sources are on the remote host (true) or local (false). 
+
+Default = false
+| *`automationedacontroller_allowed_hostnames`* | List of additional addresses to enable for user access to {EDAcontroller}.
+
+Default = empty list
+| *`automationedacontroller_controller_verify_ssl`* | Boolean flag used to verify Automation Controller's web certificates when making calls from {EDAcontroller}. Verified is true; not verified is false. 
+
+Default = false
+| *`automationedacontroller_websocket_ssl_verify`* | 
+SSL verification for the Daphne websocket used by podman to communicate from the pod to the host. Default is false to disable SSL connection as verified
```

Comment:
The description would not be exact, I suggest: 

```suggestion
SSL verification used by Decision Environments to communicate from the pod to the websocket server daphne. Default is false to disable SSL connection as verified
```

Anyway, I see this variable is not used in the installer. Ii is currently hardcoded to "no". So maybe it must be removed, can you confirm @jamesmarshall24 ?

Reply:
iirc the agreement was to fix this in 2.4-next, don't think we really need to remove this as this change should be pretty high priority. We need to secure the websocket communication.

Reply:
@Alex-Izquierdo and @jamesmarshall24 - who makes the final decision about this?

Reply:
@jself-sudoku we have a solution, so I think we can remove this variable/line from the list and then add it back to the docs on the async release where we incorporate verification again

Reply:
Okay. Thanks @jamesmarshall24. Will remove this and update the PR.

Reply:
I commented out this info with a note to add it back for a later release.

Reply:
please, don't forget to update the description anyway which is not correct, thanks.

Reply:
Just an FYI, @jamesmarshall24 and @Alex-Izquierdo : I've commented this portion out until we have this resolved. When things are ready, I can undo this action and bring the content back.